virtcontainers/network.go

```diff
@@ -511,6 +581,16 @@ func (n *NetworkNamespace) UnmarshalJSON(b []byte) error {
 			endpoints = append(endpoints, &endpoint)
 			virtLog.Infof("VhostUser endpoint unmarshalled [%v]", endpoint)
 
+		case BridgedMacvlanEndpointType:
+			var endpoint BridgedMacvlanEndpoint
+			err := json.Unmarshal(e.Data, &endpoint)
+			if err != nil {
+				return err
+			}
+
+			endpoints = append(endpoints, &endpoint)
+			virtLog.Infof("Macvlan endpoint unmarshalled [%v]", endpoint)
```

Comment:
I'm not sure what format the endpoint is in but it might be better to write this as:

```
virtLog.WithField("endpoint", endpoint).Info("Macvlan endpoint unmarshalled")
```

Reply:
^ This comment hasn't been addressed yet :)

Reply:
fixed.